catroid/src/main/AndroidManifest.xml

```diff
@@ -243,6 +243,11 @@
         <service android:name=".transfers.MediaDownloadService" />
         <service android:name="com.parrot.freeflight.service.DroneControlService" />
         <service android:name=".utils.StatusBarNotificationManager$NotificationActionService" />
-
-    </application>
+        <service
+            android:name=".ui.cloudmessaging.CustomFirebaseMessagingService">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+            </intent-filter>
+        </service>
```

Comment:
Thanks. I'll create a ticket for that. For now, it has low priority :wink:

Standalone versions are just apks with a single Catrobat program and they start directly in the stage (or so). So no IDE or Editor.
The apk generator will create apps (standalones) of each program. So if you've installed lots of standalone apks, they will all start-up and drain battery. They'll also always listen to FCM messages. I'm not sure if it's a big problem, but if we can fix it smoothly or easily, I would go with that. However, as said, low priority.